product_variant_default_code/README.rst

```diff
@@ -50,7 +53,7 @@ The automatically generated default value for the Variant reference mask will
 be `[Color]-[Size]` and so the 'default code' on the variants will be `Bl-L`,
 `Wh-L`, `Bl-XL` and `Wh-XL`.
 
-The mask value can be fully custumizedwhatever you like. You can even have
+The mask value can be fully customized whatever you like. You can even have
```

Comment:
We should add a section explaining all the new stuff added the two modes in wich the module can behave